intel_owl/settings/django.py

```diff
@@ -17,6 +17,10 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "certego_saas.ext.middlewares.LogMiddleware",  # custom
 ]
+
+SESSION_COOKIE_HTTPONLY = True
+SESSION_COOKIE_AGE = 60 * 60 * 24 * 90  # seconds * minutes * hours * days
```

Comment:
90 days for keeping the session alive? this means that, once authenticated, the session cookie is valid for 90 days and only after that time the user needs to login again? That is very bad on a security perspective. I'd like to keep this value customizable. In this way the users can decide whether to invest more in security rather than usability. It's a matter of risk analysis.
Then, I think that the default could be kept 2 weeks as the Django default. Or there are any other reasons that I am missing?

Reply:
I agree with 2 weeks. I think it's possible to make it customizable, but could required some works: first of all this is a django settings, so we need to find a way to go around it. I'm not sure is possible and if it's possible do it without a strange workaround. Also we will need a UI or something like that (we need to ask to the user the time frame he likes).